src/ddmd/root/md5.d

```diff
@@ -0,0 +1,381 @@
+// An implementation of MD5 message digest algorithm
+// Copied from Phobos std.digest.md5
+module ddmd.root.md5;
```

Comment:
Why do we need a cryptographic hash when the keys are strings and we throw away all of those digest bits anyhow?
Just using ddmd.root.hash.calcHash is fine here.

Reply:
The key is rather length string representation of the arguments so MD5 is nice shortening w/o losing the uniqueness requirement. Also we could use lower 64bits for first probe and higher 64-bit for second probe in cooko hashing.